exercises/concept/temperature/.docs/hints.md

```diff
@@ -0,0 +1,13 @@
+# General
+
+Temperature in degrees Celsius is defined as: T(°F) = T(°C) × 1.8 + 32
+
+Temperature in degrees Fahrenheit is defined as: T(°C) = (T(°F) - 32) / 1.8
```

Comment:
```suggestion
- Temperature in degrees Celsius is defined as: T(°F) = T(°C) × 1.8 + 32
- Temperature in degrees Fahrenheit is defined as: T(°C) = (T(°F) - 32) / 1.8
```

Reply:
These two hints should be task-specific hints, as they can be linked to a particular task.